tests/test_slide_thumbnail.py

```diff
@@ -68,9 +68,7 @@ def test_command_line_slide_thumbnail_output_none(sample_svs, tmp_path):
 
     assert slide_thumb_result.exit_code == 0
     assert (
-        pathlib.Path(sample_svs).parent
-        / ("slide-thumbnail")
-        / (sample_svs.stem + ".jpg")
+        pathlib.Path(sample_svs).parent / ("output") / (sample_svs.stem + ".jpg")
```

Comment:
Output doesn't need brackets here. It doesn't matter, but just looks a little odd.

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/307/commits/c19e3cc7d4ac22cd26b37ca40b0790edf71d27ce